content/docs/examples/multicluster/gke/index.md

```diff
@@ -317,7 +318,7 @@ $ kubectl label secret ${CLUSTER_NAME} istio/multiCluster=true -n ${NAMESPACE}
 ## Uninstalling
 
 The following should be done in addition to the uninstall of Istio as described in the
-[Kubernetes multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/):
+[flat-network multicluster uninstall section](/docs/setup/kubernetes/multicluster-install/vpn/):
```

Comment:
Do not use the term "flat-network" as @john-a-joyce mentioned, the term is loaded with meaning that does not apply.